src/NzbDrone.Core/Download/Clients/RQBit/ResponseModels/PostTorrentResponse.cs

```diff
@@ -0,0 +1,26 @@
+using System.Collections.Generic;
+using Newtonsoft.Json;
+
+namespace NzbDrone.Core.Download.Clients.RQBit.ResponseModels;
+
+public class PostTorrentResponse
+{
+    [JsonProperty("id")]
+    public long Id { get; set; }
+    [JsonProperty("details")]
+    public PostTorrentDetailsResponse Details { get; set; }
+    [JsonProperty("output_folder")]
+    public string OutputFolder { get; set; }
+    [JsonProperty("seen_peers")]
+    public List<string> SeenPeers { get; set; }
```

Comment:
Separate the properties and attributes for readability.

```suggestion
    [JsonProperty("id")]
    public long Id { get; set; }

    [JsonProperty("details")]
    public PostTorrentDetailsResponse Details { get; set; }

    [JsonProperty("output_folder")]
    public string OutputFolder { get; set; }

    [JsonProperty("seen_peers")]
    public List<string> SeenPeers { get; set; }
```

Reply:
This same issue is in other files as well, not going to comment on each one.